Week2/homework/index.js

```diff
@@ -0,0 +1,170 @@
+const pro = require('prompt');
+const mysql = require('mysql');
+const con = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'world',
+})
+
+
+
+const searchCapital = `select name 
+from city 
+where id 
+in (select capital from country where name = ?);`
+
+const searchRegion = `select distinct language 
+from countrylanguage 
+where countrycode 
+in(select code from country where region = ?);`
+
+const searchNumOfCities = `select distinct name 
```

Comment:
I understand what you have done giving the array length. But what if there is a lot of data, is it worth it to consume all this memory just to give the length of an array ? Maybe you can find a better solution using sql ?

Reply:
fixed! As you said I found a very simple way, thank you.